pkg/db/src/types.ts

```diff
@@ -38,6 +38,13 @@ export interface VolumeStock {
 	warehouseId: VersionedString;
 }
 
+/**
+ * Used as param(s) for adding volumes to a note where the warehouseId could be optional and a plain string (non-versioned)
+ */
+export interface VolumeStockOptionalWarehouseId extends Omit<VolumeStock, 'warehouseId'> {
+	warehouseId?: string;
+}
```

Comment:
With respect to some comments left already (being completely fine with some note transactions, temporarily, not having `warehouseId`, let's simply make the `warehouseId` optional in `VolumeStock` (this will also trickle down to `VolumeStockClient` (which is ok)